domains/letters.json

```diff
@@ -1,9 +1,10 @@
 {
     "owner": {
         "username": "keli5",
-        "email": "kel5isgod@gmail.com"
+        "discord": "keli5_letters",
+        "email": "prefer not to provide",
```

Comment:
if you dont want to provide your email but have added other socials then leave it blank